src/core/server/saved_objects/migrations/README.md

```diff
@@ -19,7 +19,7 @@ Here is the gist of what happens if an index migration is necessary:
   - Create an alias `.opensearch_dashboards` that points to `.opensearch_dashboards_1`
 - Create a `.opensearch_dashboards_2` index
 - Copy all documents from `.opensearch_dashboards_1` into `.opensearch_dashboards_2`, running them through any applicable migrations
-- # Point the `.opensearch_dashboards` alias to `.opensearch_dashboards_2`
+- Point the `.opensearch_dashboards` alias to `.opensearch_dashboards_2`
```

Comment:
Tbh these values are incorrect. Do we think we should update this to be the actual system index that we use?

Reply:
We should, but I'd generally vote for opening a new issue and PR to do so.

Reply:
Agreed, @kavilla can you open an issue to address the required changes there?

Reply:
Added: https://github.com/opensearch-project/OpenSearch-Dashboards/issues/1739